src/libs/Navigation/OnyxTabNavigator.tsx

```diff
@@ -35,25 +37,24 @@ function OnyxTabNavigator({id, selectedTab, children, ...rest}) {
             backBehavior="initialRoute"
             keyboardDismissMode="none"
             screenListeners={{
-                state: (event) => {
-                    const state = event.data.state;
+                state: (e) => {
+                    const event = e as unknown as EventMapCore<NavigationState>['state'];
```

Comment:
Why do we need `as unknown` assertion?

Reply:
It is because `screenListeners` is wrongly typed, the event for state listener should be of type `EventArg<"state", unknown, EventMapCore<NavigationState>['state']>`, while it is `EventArg<"state", unknown, unknown>`, do you have an idea how to fix it?